test/inductor/test_compiled_autograd.py

```diff
@@ -103,10 +103,15 @@ def reset():
 
 class TestCompiledAutograd(TestCase):
     def setUp(self) -> None:
+        self.exit_stack = contextlib.ExitStack()
+        self.exit_stack.enter_context(
+            config.patch("record_pre_graph_bytecode_in_traces", False)
+        )
         super().setUp()
         reset()
 
     def tearDown(self) -> None:
+        self.exit_stack.close()
```

Comment:
Curious why turning it off for compiled autograd test specifically?

Reply:
There are some bytecode hook tests which look at the generated bytecode, and its nasty to change them.